include/onnxruntime/core/graph/graph_viewer.h

```diff
@@ -51,6 +51,11 @@ class GraphViewer {
   /** Gets the path of the owning model if any **/
   const Path& ModelPath() const noexcept { return graph_->ModelPath(); }
 
+  /** Gets the external data mapping table, if any. */
+  const std::unordered_map<std::string, const void*>* ExternalDataMap() const noexcept {
```

Comment:
>std::unordered_map<std::string, const void*> [](http://example.com/codeflow?start=8&length=44)

Use InlinedHashMap with reserve() per coding standards to avoid per node allocations.